app/assets/javascripts/statistics.js

```diff
@@ -0,0 +1,88 @@
+$(function () {
+  getStatistics('current_month');
```

Comment:
@yunixon this function will be called on each page not depending if there is graphic or not.